awx_collection/plugins/lookup/tower_schedule_rrule.py

```diff
@@ -0,0 +1,235 @@
+# (c) 2020 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: tower_schedule_rrule
+    author: John Westcott IV (@john-westcott-iv)
+    version_added: "3.7"
+    short_description: Generate an rrule string which can be used for Tower Schedules
+    requirements:
+      - pytz
+      - python.dateutil
+    description:
+      - Returns a string based on criteria which represent an rule
+    options:
+      _terms:
+        description:
+          - The frequency of the schedule
+          - none - Run this schedule once
+          - minute - Run this schedule ever x minutes
+          - hour - Run this schedule every x hours
+          - day - Run this schedule ever x days
+          - week - Run this schedule weekly
+          - month - Run this schedule monthly
+        required: True
+        choices: ['none', 'minute', 'hour', 'day', 'week', 'month']
+      start_date:
+        description:
+          - The date to start the rule
+          - Used for all frequencies
+          - Format should be YYYY-MM-DD [HH:MM:SS]
+        type: str
+      timezone:
+        description:
+          - The timezone to use for this rule
+          - Used for all frequencies
+          - Format should be as US/Eastern
+          - Defaults to America/New_York
+        type: str
+      every:
+        description:
+          - The repition in months, weeks, days hours or minutes
+          - Used for all types except none
+        type: int
+      end_on:
+        description:
+          - How to end this schedule
+          - If this is not defined this schedule will never end
+          - If this is a positive integer this schedule will end after this number of occurances
+          - If this is a date in the format YYYY-MM-DD [HH:MM:SS] this schedule end after this date
+          - Used for all types except none
+        type: str
+      on_days:
+        description:
+          - The days to run this schedule on
+          - A comma seperated list which can contain values sunday, monday, tuesday, wednesday, thursday, friday
+          - Used for week type schedules
+      month_day_number:
+        description:
+          - The day of the month this schedule will run on (0-31)
+          - Used for month type schedules
+          - Can not be used with on_the parameter
+        type: int
+      on_the:
+        description:
+          - A description on when this schedule will run
+          - Two strings seperated by space
+          - First string is one of first, second, third, fourth, last
+          - Second string is one of sunday, monday, tuesday, wednesday, thursday, friday
+          - Used for month type schedules
+          - Can not be used with month_day_number parameters
+    extends_documentation_fragment: awx.awx.auth
```

Comment:
why does it extend this doc fragment? I don't think it should.

Reply:
Good catch, removed.